tests/composer/composer-php8.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
I introduce a number of specific composer files to test against various symfony builds.

Reply:
Including PHP8 is out of scope right now IMO, we should care about current versions and their support first, not the ones of the next months to come.
PHP8 is a follow up topic once we shipped the next version.

Reply:
That's why it's allowed to fail and I didn't spend time working on php8.